experiments/imit_benchmark.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+# This script finds the the mean and standard error of episode return after
+# training GAIL or AIRL on benchmark tasks.
+#
+# The benchmark tasks are defined in the CSV config file
+# `experiments/imit_benchmark_config.csv`.
+
+CONFIG_CSV="experiments/imit_benchmark_config.csv"
+EXPERT_MODELS_DIR="expert_models"
+TIMESTAMP=$(date --iso-8601=seconds)
+LOG_ROOT="output/imit_benchmark/${TIMESTAMP}"
+extra_configs=""
+extra_options=""
+
+SEEDS="0 1 2"
+
+TEMP=$(getopt -o f -l fast,gail,airl,run_name:,log_root:,file_storage: -- $@)
+if [[ $? != 0 ]]; then exit 1; fi
+eval set -- "$TEMP"
+
+while true; do
+  case "$1" in
+    # Fast mode (debug)
+    -f | --fast)
+      CONFIG_CSV="tests/data/imit_benchmark_config.csv"
+      EXPERT_MODELS_DIR="tests/data"
+      SEEDS="0"
+      extra_configs+="fast "
+      shift
+      ;;
+    --gail)
+      extra_configs+="gail "
+      shift
+      ;;
+    --airl)
+      extra_configs+="airl "
+      shift
+      ;;
+    --run_name)
+      extra_options+="--name $2 "
+      shift 2
+      ;;
+    --log_root)
+      LOG_ROOT="$2"
+      shift 2
+      ;;
+    --file_storage)
+      extra_options+="--file_storage $2 "
+      shift 2
+      ;;
+    --)
+      shift
+      break
+      ;;
+    *)
+      echo "Parsing error" >&2
```

Comment:
Consider changing to `echo "Unrecognized flag: $1" >&2`

Reply:
I changed the error message (because I don't expect us to ever reach that line), but I'm pretty sure that unrecognized flags error out on the `TEMP=$(getopt ...)` line instead.

Hard for me to verify right now because the `getopt` on my Mac is different from the `getopt` installed on Linux distriubtions.

Reply:
Yes, on Linux, an unrecognized flag errors out on the `getopt` line:
```
steven@astar:~/imitation$ ./experiments/imit_benchmark.sh --hello
getopt: unrecognized option '--hello'
```

Reply:
Ah, but we would in fact reach the `*` case if `getopt` recognizes a flag that our `case`-statement doesn't. Changing to your suggestion.